### PR TITLE
added cross build instructions

### PR DESCRIPTION
**Pull request by mcorino**

### CODE REVIEW

File: docs/src/cross_building.adoc [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+= Cross Building
+
+== Setting up a cross build environment
+
+=== Requirements
+
+Setting up a cross build development environment requires a suitable host environment to
+install and run all tooling. Currently TAOX11 (AXCIOMA) only supports mainstream Linux desktop
+environments (OpenSuSE, Debian, Fedora, CentOS, RHEL) as host environment for cross building.
```
Comment (jwillemsen) [inline docs/src/cross_building.adoc]: Remove `only` and `Debian`?

File: docs/src/cross_building.adoc [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+All development tool prerequisites that apply for building TAOX11 (AXCIOMA) for the host platform
+itself also apply for cross building.
+In addition a cross build toolchain for the intended target environment needs to be installed. Typically
+such toolchain packages include host compiler, linker, librarian etc. binaries as well as a target system
+root (sysroot) SDK providing target versions of libraries etc.
+
+=== Installing
+
+Installing the TAOX11 (AXCIOMA) source code for the cross build is identical to the process for a host
+build. Simply use Git to clone the base axcioma.git repository and use the `bin/brix11 bootstrap` command
```
Comment (jwillemsen) [inline docs/src/cross_building.adoc]: git

File: docs/src/cross_building.adoc [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+to finish the installation.
+Optionally a second code tree can be installed to provide a complete host build of TAOX11 (AXCIOMA). See
+the information on <<Configuration,configuration>> for more details.
+
+In addition it might be necessary to install target versions of required libraries that are not included
+in the cross build toolchain's sysroot like the RTI NDDS version for your target, a Xerces-C library version
```
Comment (jwillemsen) [inline docs/src/cross_building.adoc]: RTI Connext DDS

File: docs/src/cross_building.adoc (hunk 2 above)
Comment (mcorino) [reply docs/src/cross_building.adoc]: The product is "Git" the tool is "git". I am referring to the product here.

**Pull request merged**